config/initializers/assets.rb

```diff
@@ -53,53 +50,62 @@
                 password_strength
                 proxy_status)
 
-stylesheets = %w( )
-
-# Add the fonts path
-config.assets.paths << Rails.root.join('vendor', 'assets', 'fonts')
-
-# Precompile additional assets
-config.assets.precompile << /\.(?:svg|eot|woff|ttf)$/
+  stylesheets = %w( )
 
-config.assets.precompile += javascript.map { |js| js + '.js' } + stylesheets + %w(background-size.htc)
+  # Add the fonts path
+  config.assets.paths << Rails.root.join('vendor', 'assets', 'fonts')
 
-# Defaults to Rails.root.join("public/assets")
-# config.assets.manifest = YOUR_PATH
+  # Precompile additional assets
+  config.assets.precompile << /\.(?:svg|eot|woff|gif|ttf)$/
```

Comment:
I've added gif here as patternfly contains a few, and they have to be precompiled